src/main.js

```diff
@@ -10,7 +10,7 @@ function v86(bus, wasm)
     this.running = false;
 
     /** @type {boolean} */
-    this.stopped = false;
+    this.stopping = false;
```

Comment:
Renaming this was not necessary, so I put it into a separate commit (with explanation) so it would be easier to revert, if desired.

Reply:
`stopping` is a better name, thanks.